src/processors/default/processMempool.ts

```diff
@@ -0,0 +1,108 @@
+import _ from 'lodash';
+
+import { Table } from '../../db/db';
+import { pendingMempoolInsertMessages, pendingMempoolUpdateMessages } from '../../triggers/messages';
+import { PendingMempoolMessage, CrdtState } from '../../types/message';
+import { Clients, Processor } from '../../types/processor';
+
+function categorizeMessages
+(
+  messages: PendingMempoolMessage[],
+):
+  { results: any, crdtUpdates: CrdtState[] } {
+  const entityUpdates: { [id: string]: any } = {};
+  const crdtUpdates: CrdtState[] = [];
+
+  // NOTE: messages can be partially processed. If a column in a message is stale, it will be ignored but the rest of the message
+  // will still be processed. 
```

Comment:
```suggestion
  // NOTE: messages with stale columns are still fully processed. If a column in a message is stale, it will not be applied, but the message is still processed
```